### PR TITLE
Tests/celia test enumerables

### PR DESCRIPTION
**Pull request by Celiamg87**

### CODE REVIEW

File: BusinessCore.Tests/Celia_FunctionalProgramingTests.cs [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+        public Celia_FunctionalProgramingTests()
+        {
+            this.vehicleStorage = new ArrayVehicleStorage();
+        }
+
+        [TestMethod]
+        public void get_the_enrollment_and_horsePower_of_the_vehicles_ordered_descending_by_horsePower()
+        {          
+
```
Comment (hossmi) [inline BusinessCore.Tests/Celia_FunctionalProgramingTests.cs]: Please, add vehicles variable of type IVehicle[].

Comment (Celiamg87) [reply BusinessCore.Tests/Celia_FunctionalProgramingTests.cs]: Compile solution

File: BusinessCore.Tests/Celia_FunctionalProgramingTests.cs [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+using BusinessCore.Tests.Services;
+using CarManagement.Core.Models;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace BusinessCore.Tests
+{
+    [TestClass]
+    public class Celia_FunctionalProgramingTests
+    {
+        private readonly ArrayVehicleStorage vehicleStorage;
```
Comment (hossmi) [inline BusinessCore.Tests/Celia_FunctionalProgramingTests.cs]: please, the test must use Celia_ArrayVehicleStorage

File: BusinessCore.Tests/BusinessCore.Tests.csproj [hunk 3]
```diff
@@ -59,13 +59,15 @@
     <Compile Include="Services\ArrayEnrollmentProvider.cs" />
     <Compile Include="EpicSimpleVehicleTests.cs" />
     <Compile Include="Services\ArrayVehicleStorage.cs" />
+    <Compile Include="Services\Celia_ArrayVehicleStorage.cs" />
```
Comment (hossmi) [inline BusinessCore.Tests/BusinessCore.Tests.csproj]: Please, rename to CeliaArrayVehicleStorage.cs

File: BusinessCore.Tests/BusinessCore.Tests.csproj [hunk 4]
```diff
@@ -59,13 +59,15 @@
...
     <Compile Include="Services\SingleEnrollmentProvider.cs" />
     <Compile Include="Negassert.cs" />
     <Compile Include="NumberToolsTests.cs" />
     <Compile Include="Properties\AssemblyInfo.cs" />
     <Compile Include="JVBBTests.cs" />
     <Compile Include="CeliaVehicleTest.cs" />
     <Compile Include="FunctionalProgramingTests.cs" />
+    <Compile Include="Celia_FunctionalProgramingTests.cs" />
```
Comment (hossmi) [inline BusinessCore.Tests/BusinessCore.Tests.csproj]: Pklease, rename to CeliaFunctionalProgramingTests.cs

**Pull request closed without merging**